# Generated by Django 5.2.4 on 2025-07-19 08:02

import django.db.models.deletion
from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('watergis', '0002_waterqualitystation_measurement_end_date_and_more'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.AddField(
            model_name='alert',
            name='resolved_at',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='alert',
            name='resolved_by',
            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='resolved_alerts', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='alert',
            name='severity',
            field=models.CharField(choices=[('low', 'Low'), ('medium', 'Medium'), ('high', 'High'), ('critical', 'Critical')], default='medium', max_length=20),
        ),
        migrations.AddField(
            model_name='watersample',
            name='conductivity',
            field=models.FloatField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='watersample',
            name='total_dissolved_solids',
            field=models.FloatField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='alert',
            name='alert_type',
            field=models.CharField(choices=[('water_quality', 'Water Quality Issue'), ('equipment', 'Equipment Malfunction'), ('maintenance', 'Maintenance Required'), ('data_gap', 'Data Gap Detected'), ('threshold_exceeded', 'Threshold Exceeded')], max_length=100),
        ),
    ]
